sdk/core/azure-core/tests/test_basic_transport.py

```diff
@@ -96,27 +92,6 @@ def test_url_join():
     assert _urljoin('devstoreaccount1/', 'testdir/') == 'devstoreaccount1/testdir/'
 
 
-def test_http_client_response():
```

Comment:
i'm thinking that instead of migrating all of these tests into the little testserver_tests folder, why don't we get rid of the testserver_tests folder? What are your thoughts?

I think it's worth it to start now, esp if we're going to start moving more and more to testserver

Reply:
Hah. I was thinking about the opposite. :)

I noticed that some tests don't need test server at all. So in addition to keeping the testserver_tests folder, I was thinking moving some of your tests out of the folder to make them run faster.

Your ideas?

Reply:
Ah I see your point. I'm down with that, I don't feel too strongly!